src/seedu/addressbook/AddressBook.java

```diff
@@ -226,6 +221,15 @@ public static void main(String[] args) {
      * ====================================================================
      */
 
+    private static void processUserInput() {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.